deps/verifier/src/tdx/quote.rs

```diff
@@ -506,7 +509,60 @@ pub async fn ecdsa_quote_verification(quote: &[u8]) -> Result<()> {
         }
     }
 
-    Ok(())
+    let tcb_status_desc = SgxQlQvResultWrapper(quote_verification_result).as_str();
+
+    let advisory_ids: Vec<Value> = get_sa_list(&supp_data.sa_list).iter()
+        .map(|s| Value::String(s.to_string()))
+        .collect();
+
+    let mut claims_map = Map::new();
```

Comment:
we may want `collateral_expiration_status` as  a token field too

Reply:
Updated claims to include it